src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -527,6 +621,23 @@ private static void AppendTextFromAttribute(FormatterState state, XElement eleme
                 new SymbolDisplayPart(kind, symbol: null, text: TrimCrefPrefix(crefValue)));
         }
 
+        internal static IEnumerable<SymbolDisplayPart> TypeParameterRefToSymbolDisplayParts(
+            string crefValue, ISymbol typeResolutionSymbol, int position, SemanticModel semanticModel, SymbolDisplayFormat format = null)
```

Comment:
format should not be nullable/optional.

Reply:
it was in CrefToSymbolDisplayParts, so we need to change both, right?

Reply:
you can if you want.  but this one should definitely change.

Reply:
done